tensilelite/Tensile/SolutionStructs.py

```diff
@@ -3451,6 +3459,16 @@ def subCheckLdsBlockSizePerPad(tc, idx):
           ldsBiasMaxElements = max(ldsBiasMaxElements, state["MacroTile0"] * dataType.numBytes())
       ldsNumBytes = max(ldsNumBytes, state["LdsOffsetBias"] + ldsBiasMaxElements)
 
+    state["LdsBytesNoAmax"] = ldsNumBytes
+    if state["ProblemType"]["OutputAmaxD"]:
+      # used in reduce inter wave
+      # 4 data * half_wave_num * amax bytePerE
+      num_workItems = state["NumThreads"]
+      half_wave_size = state["WavefrontSize"] // 2
+      amaxBPE = state["ProblemType"]["DataTypeAmaxD"].numBytes()
+      ldsAmaxDBytes = 4 * (num_workItems // half_wave_size) * amaxBPE
+      ldsNumBytes += ldsAmaxDBytes
+
     state["LdsNumBytes"] = ldsNumBytes
```

Comment:
AmaxD reduction needs some extra LDS. We keep the LDS size can't be used by AmaxD, and use it as the LDSOffset in reudction section.